parlai/core/params.py

```diff
@@ -1043,10 +1054,10 @@ def _kwargs_to_str_args(self, **kwargs):
                 string_args.append(last_option_string)
```

Comment:
@stephenroller are you referring to this also being changed to ```_value2argstr(last_option_string)``` ?

Reply:
No, I'm referring to the `string_args.append(str(value))` down on lines 1092 and 1095

Reply:
Unsure how I missed that, thanks for pointing out. Fixed them in latest push